### PR TITLE
Add note on Kibana OAuth refresh

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1791837#c29

### CODE REVIEW

**Comment by mburke5678** [comment]
@jcantrill @sichvoge PTAL

**Comment by mburke5678** [comment]
Added associated note to release notes: https://github.com/openshift/openshift-docs/pull/22581/

**Comment by mburke5678** [comment]
@anpingli PTAL

**Comment by mburke5678** [comment]
@anpingli Please take a look at this small change.

**Review by anpingli — COMMENTED** [review]
LGTM

File: modules/cluster-logging-kibana-interface-launch.adoc [hunk 1]
```diff
@@ -24,3 +24,8 @@ The Kibana interface launches. You can now:
 +
 Use and configuration of the Kibana interface is beyond the scope of this documentation. For more information,
 on using the interface, see the link:https://www.elastic.co/guide/en/kibana/5.6/connect-to-elasticsearch.html[Kibana documentation].
+
+[NOTE]
+====
+If you get a *security_exception* error in the Kibana console and cannot access your Kibana indices, you might have an expired OAuth token. If you see this error, log out of the Kibana console, then log back in. This refreshes your OAuth tokens and you should be able to access your indices.
```
Comment (bergerhoffer) [inline modules/cluster-logging-kibana-interface-launch.adoc:30]: then log back in -> and then log back in

**Review by bergerhoffer — COMMENTED** [review]
Just one minor thing

**Pull request merged**